cg/constants/priority.py

```diff
@@ -12,3 +12,9 @@ class SlurmQos(StrEnum):
     LOW: str = "low"
     NORMAL: str = "normal"
     HIGH: str = "high"
+
+
+RSYNC_ACCOUNT_TO_QOS = {
```

Comment:
Sorry for doing this review in a drive by fashion
```suggestion
SLURM_ACCOUNT_TO_QOS = {
```
Maybe the link between Slurm account and prio should be more general